lib/rack/tracker/criteo/criteo.rb

```diff
@@ -0,0 +1,31 @@
+class Criteo <  Rack::Tracker::Handler
+  class Event < OpenStruct
+    def write
+      to_h.to_json
+    end
+  end
+
+  self.position = :body
+
+  def tracker_options
```

Comment:
agreed, but then look into the last changes in #15 to make this more generic. I see a chance here of making them into Event objects and prepend them into the events array.